### ISSUE TITLE
Woocommerce 4.3.1 seems to change edit post font style in WP 5.5 RC1

### ISSUE THREAD

**Issue by dariaknl** [woocommerce/woocommerce#27161]
Please provide us with the information requested in this bug report. Without these details, we won't be able to fully evaluate this issue. 
Bug reports lacking detail, or for any other reason than to report a bug, may be closed without action.

**Prerequisites (mark completed items with an [x]):**
- [x] I have have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**
I have not found this issue here. If it is known, please provide info how/when it is going to be fixed, as it influences all metaboxes which are shown on edit post/page. 
WP 5.5 release candidate 1: when you activate Woocommerce 4.3.1, the edit post fonts are changed (it seems to change to comic sans).

**Expected behavior**
Edit post page should not be affected.

**Actual behavior**
<img width="1604" alt="Screenshot 2020-07-29 at 11 48 44" src="https://user-images.githubusercontent.com/19681708/88785700-d0193f00-d191-11ea-9f55-26d47b97cd3a.png">

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. Install WP 5.5 RC1 via WordPress Beta Tester plugin.
2. Install and activate Woocommerce 4.3.1.
3. Click to create new post.

**Screenshots**

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
We use the [WooCommerce System Status Report](https://docs.woocommerce.com/document/understanding-the-woocommerce-system-status-report/) to help us evaluate the issue. 
Without this report we won't be able to fully evaluate this issue.
<details>
```
The System Status Report is found in your WordPress admin under **WooCommerce > Status**. 
Please select “Get system report”, then “Copy for support”, and then paste it here.
```
</details>

**Comment by juliaamosova** [comment]
Hi @dariaknl,

Thank you for taking the time to report this bug, we really appreciate your help. I can reproduce it on my end. 

I first thought that this is not WooCommerce related but I confirmed that the issue happens only when WooCommerce is activated indeed. 

Before WooCommerce is activated using WordPress 5.5-RC1-48681:

![before](https://user-images.githubusercontent.com/19143190/88817675-d0aed700-d18b-11ea-9ca7-8488e0ded912.jpg)

After WooCommerce is activated using WordPress 5.5-RC1-48681:

![after](https://user-images.githubusercontent.com/19143190/88817717-db696c00-d18b-11ea-819b-f36b8d6e1e93.jpg)

We’re going to work on this bug as soon as possible.

**Comment by juliaamosova** [comment]
Noting that I am also getting the font switch on the System Status Report:

<img width="1440" alt="Screen Shot 2020-07-29 at 11 19 29 AM" src="https://user-images.githubusercontent.com/19143190/88819054-76167a80-d18d-11ea-8f47-fd5407e15664.png">

**Comment by claudiosanches** [comment]
Hello @dariaknl,

Sounds like this is related to the PHP warnings on the screen.

Try put the follow lines to your `wp-config.php` file and check if the problem is fixed.

```php
@ini_set( 'display_errors', 0 );
```

**Comment by claudiosanches** [comment]
I just confirmed, when the messages are disabled the font is restored.
Closing since this is generated by WooCommerce Blocks and there's a fix for it that will be included into 4.4.0 Beta 2.
